### PR TITLE
[Vectorize] update docs for Vectorize V2

### PR DESCRIPTION
**Pull request by sejoker**
### Summary
Vectorize V2 changes:
- use of Wrangler prerelease version
- wrangler commands for metadata index operations
- updated V2 limits
- added recommendation for vectors bulk upload

### CODE REVIEW

File: content/vectorize/best-practices/insert-vectors.md [hunk 1]
```diff
@@ -117,6 +117,11 @@ Refer to [Vectorize API](/vectorize/reference/client-api/) for additional exampl
 
 ### wrangler CLI
 
+{{<Aside type="note" header="Cloudflare API rate limit">}}
+
+Please use a maximum of 5000 vectors per embeddings.ndjson file to prevent the global [rate limit](/fundamentals/api/reference/limits/) for the Cloudflare API.
```
Comment (netgusto) [inline content/vectorize/best-practices/insert-vectors.md]: I think wrangler can read a bigger file, but will batch lines up to 5K

File: content/vectorize/get-started/intro.md [hunk 2]
```diff
@@ -18,7 +18,13 @@ This guide will instruct you through:
 
 ## Prerequisites
 
-To continue:
+{{<Aside type="note" header="Workers Paid plan required">}}
+
+Hyperdrive is available to all users on the [Workers Paid plan](/workers/platform/pricing/#workers).
```
Comment (netgusto) [inline content/vectorize/get-started/intro.md]:
```suggestion
Vectorize is available to all users on the [Workers Paid plan](/workers/platform/pricing/#workers).
```

File: content/vectorize/reference/client-api.md [hunk 3]
```diff
@@ -86,6 +86,46 @@ const details = await env.YOUR_INDEX.describe();
...
+### Create Metadata Index
+
+Enable metadata filtering on the specified property. Limited to 10 properties.
+
+{{<render file="_vectorize-wrangler-version.md">}}
+
+Run the following `wrangler vectorize` command:
+
+```sh
+$ wrangler vectorize create-metadata-index <name>
```
Comment (netgusto) [inline content/vectorize/reference/client-api.md]: There should be an index type in the command arguments as well (string, number, boolean)

Comment (netgusto) [reply content/vectorize/reference/client-api.md]: \+ the vectorize index name

File: content/vectorize/reference/client-api.md [hunk 4]
```diff
@@ -86,6 +86,46 @@ const details = await env.YOUR_INDEX.describe();
...
+### Delete Metadata Index
+
+Allow Vectorize to delete the specified metadata index.
+
+{{<render file="_vectorize-wrangler-version.md">}}
+
+Run the following `wrangler vectorize` command:
+
+```sh
+$ wrangler vectorize delete-metadata-index <name>
```
Comment (netgusto) [inline content/vectorize/reference/client-api.md]: \+ the vectorize index name

File: content/vectorize/reference/client-api.md [hunk 5]
```diff
@@ -86,6 +86,46 @@ const details = await env.YOUR_INDEX.describe();
...
+### List Metadata Indexes
+
+List metadata properties on which metadata filtering is enabled.
+
+{{<render file="_vectorize-wrangler-version.md">}}
+
+Run the following `wrangler vectorize` command:
+
+```sh
+$ wrangler vectorize list-metadata-index <name>
```
Comment (netgusto) [inline content/vectorize/reference/client-api.md]: \+ the vectorize index name

File: content/vectorize/get-started/intro.md (hunk 2 above)
Comment (kodster28) [inline content/vectorize/get-started/intro.md]:
```suggestion
Vectorize is available to all users on the [Workers Paid plan](/workers/platform/pricing/#workers).
```

File: content/vectorize/reference/metadata-filtering.md [hunk 6]
```diff
@@ -6,6 +6,13 @@ weight: 6
 
 # Metadata Filtering
 
+{{<Aside type="note" header="Enable metadata filtering">}}
+
+Vectorize V2 requires to define metadata properties that will support filtering. Please refer to [Vectorize API](/vectorize/reference/client-api/#create-metadata-index) for details.
```
Comment (netgusto) [inline content/vectorize/reference/metadata-filtering.md]: Are we refering to this version as V2? Perhaps beta 2?

File: content/vectorize/reference/metadata-filtering.md [hunk 7]
```diff
@@ -6,6 +6,13 @@ weight: 6
...
+[Namespaces](/vectorize/best-practices/insert-vectors/#namespaces) support metadata filtering out of the box.
```
Comment (kodster28) [inline content/vectorize/reference/metadata-filtering.md]:
```suggestion
[Namespaces](/vectorize/best-practices/insert-vectors/#namespaces) support metadata filtering by default.
```

**Review by kodster28 — APPROVED** [review]
Two small changes

File: content/vectorize/best-practices/insert-vectors.md (hunk 1 above)
Comment (garvit-gupta) [reply content/vectorize/best-practices/insert-vectors.md]: That would be correct. Wrangler can split a large file into batches of 5000 for Vectorize V2 and process each batch as a separate insert request. Each batch however must be less than 5000 vectors and less than 128 MB for the insert request to succeed.

File: content/vectorize/reference/client-api.md [hunk 8]
```diff
@@ -86,6 +86,46 @@ const details = await env.YOUR_INDEX.describe();
 
 Retrieves the configuration of a given index directly, including its configured `dimensions` and distance `metric`.
 
+### Create Metadata Index
```
Comment (garvit-gupta) [inline content/vectorize/reference/client-api.md]: Should we include the index info command too?

Comment (sejoker) [reply content/vectorize/reference/client-api.md]: > Should we include the index info command too?

added.

File: content/vectorize/reference/metadata-filtering.md (hunk 7 above)
Comment (sejoker) [reply content/vectorize/reference/metadata-filtering.md]: resolved.

File: content/vectorize/get-started/intro.md (hunk 2 above)
Comment (sejoker) [reply content/vectorize/get-started/intro.md]: resolved.

File: content/vectorize/reference/client-api.md [hunk 9]
```diff
@@ -86,6 +86,60 @@ const details = await env.YOUR_INDEX.describe();
...
+### List Metadata Indexes
+
+List metadata properties on which metadata filtering is enabled.
+
+{{<render file="_vectorize-wrangler-version.md">}}
+
+Run the following `wrangler vectorize` command:
+
+```sh
+$ wrangler vectorize list-metadata-index <index-name> --property-name='some-prop' --type='string'
```
Comment (garvit-gupta) [inline content/vectorize/reference/client-api.md]: `list-metadata-index` does not need the `--property-name` or `--type` params

File: content/vectorize/reference/client-api.md [hunk 10]
```diff
@@ -86,6 +86,60 @@ const details = await env.YOUR_INDEX.describe();
...
+### Delete Metadata Index
+
+Allow Vectorize to delete the specified metadata index.
+
+{{<render file="_vectorize-wrangler-version.md">}}
+
+Run the following `wrangler vectorize` command:
+
+```sh
+$ wrangler vectorize delete-metadata-index <index-name> --property-name='some-prop' --type='string'
```
Comment (garvit-gupta) [inline content/vectorize/reference/client-api.md]: `delete-metadata-index`does not need the `--type` param

**Pull request merged**